build.config

```diff
@@ -0,0 +1,61 @@
+# -*- python -*-
+# ex: set syntax=python:
+
+# The base URL for the repo
+BASEURL = 'https://github.com/mcallegari/qlcplus'
+# Stub to add to the base URL to pull from
+REPOSTUB = '.git'
+# Stub to add to the base URL to get revision links
+REVLINKSTUB = '/commit/%s'
+
+# Title, 'project' is appended to the title.
+TITLE = "QLC+"
+# URL of the project
+URL = "http://qlcplus.sourceforge.net/index.shtml"
+# URL of the buildbot
+BUILDBOTURL = "https://qlcplusbuildbot.openlighting.org/"
+
+# IRC server
+IRCSERVER = "irc.freenode.org"
+# IRC bot base name
+IRCBOTBASENAME = "QLCplus-buildbot"
+# Main IRC channel
+# Bot IRC channel
+#IRCBOTCHANNEL = "#qlcplus-build"
+
+# The port the slaves connect to
+SLAVE_PORT = 9989
+
+CPP_LINT_ARGS = '--filter=-legal/copyright,-readability/streams,-runtime/arrays'
```

Comment:
I've commented it out and tweaked master.cfg so it skips it if the option isn't present.